mne/_freesurfer.py

```diff
@@ -57,44 +57,68 @@ def _import_nibabel(why='use MRI files'):
     return nib
 
 
-def _mri_orientation(img, orientation):
-    """Get MRI orientation information from an image.
+def _reorient_image(img, axcodes='RAS'):
+    """Reorient an image to a given orientation.
 
     Parameters
     ----------
     img : instance of SpatialImage
         The MRI image.
+    axcodes : tuple | str
+        The axis codes specifying the orientation, e.g. "RAS".
+        See :func:`nibabel.orientations.aff2axcodes`.
+
+    Returns
+    -------
+    img_data : ndarray
+        The reoriented image data.
+    vox_ras_t : ndarray
+        The new transform from the new voxels to surface RAS.
+
+    Notes
+    -----
+    .. versionadded:: 0.24
+    """
+    import nibabel as nib
+    orig_data = np.array(img.dataobj).astype(np.float32)
+    # reorient data to RAS
+    ornt = nib.orientations.axcodes2ornt(
+        nib.orientations.aff2axcodes(img.affine)).astype(int)
+    ras_ornt = nib.orientations.axcodes2ornt(axcodes)
+    ornt_trans = nib.orientations.ornt_transform(ornt, ras_ornt)
+    img_data = nib.orientations.apply_orientation(orig_data, ornt_trans)
```

Comment:
Can you verify in the nibabel code that this just ends up being a view on the data? slices are always views, and I think `.transpose`s are, too (?), and in theory this is all that should be needed and nibabel is generally very well coded so probably does this... but please do check so we can all sleep better at night.

If it doesn't create a view, we should maybe create an issue about enhancing their code to return a view if possible. (This is should be better memory--wise).

Reply:
https://github.com/nipy/nibabel/blob/62aea04248e70d7c4529954ca41685d7f75a0b1e/nibabel/orientations.py#L154:L172

Looks nothing here makes a copy.

Reply:
So I think we're good